packages/@lwc/signals/package.json

```diff
@@ -40,5 +40,8 @@
                 ]
             }
         }
+    },
+    "dependencies": {
+        "@lwc/shared": "8.2.0"
```

Comment:
You may as well make this a `devDep` so that the `SIGNAL_SYMBOL` is just embedded in the `dist` file for `@lwc/signals`.